### PR TITLE
feat: load whitelisted packages only

### PR DESCRIPTION
**Pull request by sidohaakma**

### CODE REVIEW

File: armadillo/src/main/resources/application.yml [hunk 1]
```diff
@@ -18,11 +21,16 @@ datashield:
...
           seed: 342325352
 #          foo: bar
-    - name: exposome
-      environment: exposome
+#    - name: exposome
+#      environment: exposome
+#      whitelist:
+#        - dsBase
+#        - dsExposome
+    - name: omics
```
Comment (tommydeboer) [inline armadillo/src/main/resources/application.yml:29]: Should we comment out the `omics` profile too?

File: r/src/main/java/org/molgenis/r/service/PackageServiceImpl.java [hunk 2]
```diff
@@ -68,6 +72,18 @@ public PackageServiceImpl(REXPParser rexpParser) {
     }
   }
 
+  @Override
+  public void loadPackages(RConnection connection, Set<String> pkgs) {
+    try {
+      String packages = String.format("\"%s\"", String.join("\",\"", pkgs));
+      LOGGER.debug("Loading packages [ {} ]", packages);
+      connection.eval("library(" + packages + ")");
+      LOGGER.debug("Successfully loaded packages [ {} ]", packages);
```
Comment (tommydeboer) [inline r/src/main/java/org/molgenis/r/service/PackageServiceImpl.java:81]: Maybe demote this one to `trace`

Comment (tommydeboer) [reply r/src/main/java/org/molgenis/r/service/PackageServiceImpl.java:81]: Or remove it

**Review by tommydeboer — CHANGES_REQUESTED** [review]

File: armadillo/src/main/resources/application.yml (hunk 1 above)
Comment (sidohaakma) [reply armadillo/src/main/resources/application.yml:29]: Yes, it needs to be done in de docker-compose as well. I will do this.

File: r/src/main/java/org/molgenis/r/service/PackageServiceImpl.java (hunk 2 above)
Comment (sidohaakma) [reply r/src/main/java/org/molgenis/r/service/PackageServiceImpl.java:81]: I will demote it to trace.

File: README.md [hunk 3]
```diff
@@ -30,3 +30,20 @@ To spin up your own server on a laptop, you can run `docker-compose up` in the d
 
 Alternatively for [kubernetes](https://k8s.io) deployments look at the [helm chart](https://github.com/molgenis/molgenis-ops-helm/tree/master/charts/molgenis-armadillo). 
 Or for virtual server deployment look at the [ansible playbook](https://galaxy.ansible.com/molgenis/armadillo).
+
+### Development
+You need to start several backend services in other to be able to develop in the Armadillo.
```
Comment (tommydeboer) [inline README.md:35]:
```suggestion
You need to start several backend services in order to be able to develop in the Armadillo.
```

**Pull request merged**